mentat/terminal/output.py

```diff
@@ -29,13 +31,22 @@ def print_stream_message(message: StreamMessage):
     end = "\n"
     color = None
     flush = False
+    session_context = SESSION_CONTEXT.get()
+    if session_context.config.theme is not None:
```

Comment:
I prefer `Themes.get(config.theme)` to the if statement. It will be `None` if `config.theme` is anyway.

Reply:
Redid getting the theme based on the suggestion and added a validation check on the theme name.